jdi-light/src/main/java/com/epam/jdi/light/common/Property.java

```diff
@@ -3,7 +3,11 @@
 public enum Property {
 
     TIMEOUT_WAIT_ELEMENT("timeout.wait.element", "^[1-9][0-9]{1,2}$|^\\d$",
-            "Value of 'timeout.wait.element' must be an integer from 0 to 999.");
+            "Value of 'timeout.wait.element' must be an integer from 0 to 999."),
+    SCREENS_FOLDER("screens.folder", "^(?:[a-zA-Z]\\:|\\\\\\\\[\\w\\.]+\\\\[\\w.$]+)\\\\(?:[\\w]+\\\\)*\\w([\\w.])+$", "'screens.folder' must contain the correct folder path"),
```

Comment:
Since you store the name of property ("screens.folder") in this class, use it in WebSettings.class inside init() method.
Change 
> fillAction(p -> SCREEN_PATH = p, "screens.folder");
 to
> fillAction(p -> SCREEN_PATH = p, SCREENS_FOLDER.getName());

Reply:
Done